# Generated with epl_ros_generator.py

# openPOWERLINK variables:
string shm_name
uint16 pid

# xap.h variables:
uint8 CN1_DigitalInput_00h_AU8_DigitalInput
uint8 CN32_DigitalInput_00h_AU8_DigitalInput
uint8 CN110_DigitalInput_00h_AU8_DigitalInput
